ca/certificate-authority_test.go

```diff
@@ -165,13 +166,14 @@ type mockSA struct {
 	certificate core.Certificate
 }
 
-func (m *mockSA) AddCertificate(der []byte, _ int64) (string, error) {
+func (m *mockSA) AddCertificate(ctx context.Context, der []byte, _ int64) (string, error) {
 	m.certificate.DER = der
 	return "", nil
 }
 
 var caKey crypto.Signer
 var caCert *x509.Certificate
+var contxt = context.Background()
```

Comment:
In the blog post at https://blog.golang.org/context, they just inline multiple calls to `context.Background()`. From reading the documentation it sounds like either is fine, but picking an unusual misspelling of context to work around scoping is a net loss. I think we should just inline `context.Background()` here.